packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -659,6 +717,7 @@ const runGenerator = (context, {outputDir, templateVersion, verbose}) => {
     // downloading from NPM or copying from the template bundle folder.
     const tmp = fs.mkdtempSync(p.resolve(os.tmpdir(), 'extract-template'))
     const packagePath = p.join(tmp, 'package')
+    const extensionDir = p.join(outputDir, 'extension-sample')
```

Comment:
Do we want to plop all the "extracted" application extension projects into a shared folder. Like `app/application-extensions` or something along those lines?